bravado/mapping/validate.py

```diff
@@ -50,7 +41,7 @@ def validate_array(spec, value):
     :param spec: spec for an 'array' type in dict form
     :type value: list
     """
-    jsonschema.validate(value, spec)
+    Swagger20RequestValidator(spec).validate(value)
```

Comment:
This is a great change because it won't call `check_schema()` any more as well, which ends up being relatively expensive.